linux/Dockerfile

```diff
@@ -1,78 +1,52 @@
-ARG DEBIAN_VERSION=bookworm
```

Comment:
Could you please restore these? 
The args are there so we can control the build images on local systems (different base image and such)